haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
+	Fabien Fulhaber"
+LICENSE="custom"
+REVISION="1"
+srcGitRev="ca884c1af8bee25d71389df85001757bef4a3a6a"
```

Comment:
```diff
-srcGitRev="ca884c1af8bee25d71389df85001757bef4a3a6a"
+srcGitRev="69d6a5ada4fa76b0e900face8f6eef1d7ac48119"
```
The srcGitRev you have right now is the one from `haiku-apps/2dphysicsdemo/2dphysicsdemo-22.5.98.recipe` :smile:

Reply:
@fbrosson Oops! Must have forgotten to change it.